site2/tools/python-doc-gen.sh

```diff
@@ -26,6 +26,8 @@ ROOT_DIR=$(git rev-parse --show-toplevel)
 # so that Pdoc can import the module
 find $ROOT_DIR -name CMakeCache.txt | xargs rm -f
 find $ROOT_DIR -name CMakeFiles | xargs rm -rf
+find $ROOT_DIR -name PulsarApi.pb.h | xargs rm -rf
```

Comment:
are these 2 lines related to swagger?

Reply:
Not directly related to swagger, but these impede the building of the website.